eslint.js

```diff
@@ -125,12 +132,20 @@ async function run() {
         return;
     }
 
+    const current = process.cwd();
     const files = await gitChangedFiles(baseRef, '.');
+    const shouldRunAll = runAllIfChanged.some(name =>
+        files.some(file => path.relative(current, file) === name),
+    );
     const validExt = ['.js', '.jsx', '.mjs', '.ts', '.tsx'];
-    const jsFiles = files.filter(file => validExt.includes(path.extname(file)));
+    const jsFiles = shouldRunAll
+        ? // Get all files
+          ['.']
```

Comment:
This is resolved inside the CWD, right?

Reply:
Is eslint smart enough to know what extensions to look for?

Reply:
yup, eslintconfig specifies what extensions are allowed